docs/docs/reference/new-types/dependent-function-types.md

```diff
@@ -11,13 +11,13 @@ trait Entry { type Key; val key: Key }
 def extractKey(e: Entry): e.Key = e.key          // a dependent method
 val extractor: (e: Entry) => e.Key = extractKey  // a dependent function value
 //            ║   ⇓ ⇓ ⇓ ⇓ ⇓ ⇓ ⇓   ║
-//            ║     Dependent     ║
-//            ║   Function Type   ║
-//            ╚═══════════════════╝
+//            ║     Dependent    ║
+//            ║   Function Type  ║
+//            ╚═══════════════╝
```

Comment:
It looks like the ascii art got broken here ?

Reply:
oh wow, it looked broken on my phone and I tried to fix it :D
I'll revert this section, I guess most people are looking at the website from a PC